### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by MaisClement**
Hi,

This is just a little update of the french translation 😄

### CODE REVIEW

File: Nearby Sharing Windows/Resources/values-fr/strings.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<string name="app_name">Partage de proximité</string>
+	<string name="app_name">Partage de proximité (Windows)</string>
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-fr/strings.xml:3]: "Windows" has been removed from the name
```suggestion
	<string name="app_name">Partage de proximité</string>
```

File: Nearby Sharing Windows/Resources/values-fr/strings.xml [hunk 2]
```diff
@@ -40,19 +40,45 @@
...
+	<string name="open_discord_summary">Rejoindre le serveur Discord</string>
+
+	<string name="preference_appearance_title">Interface et apparence</string>
+	<string name="preference_appearance_summary">Langage, Thème sombre</string>
+
+	<string name="force_dark_mode_title">Thème sombre</string>
+	<string name="force_dark_mode_summary">Forcer le thème sombre</string>
+
+	<string name="switch_language_title">Langue</string>
+	<string name="switch_language_summary">Changer la langue de l'appli</string>
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-fr/strings.xml:67]:
```suggestion
	<string name="switch_language_summary">Changer la langue de l’appli</string>
```
or
```suggestion
	<string name="switch_language_summary">Changer la langue de l\'appli</string>
```

File: Nearby Sharing Windows/Resources/values-fr/strings.xml [hunk 3]
```diff
@@ -40,19 +40,45 @@
...
+
+	<string name="preference_cdp_title">Connectivité</string>
+	<string name="preference_cdp_summary">Configuration pour l'envoi et la réception</string>
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-fr/strings.xml:70]:
```suggestion
	<string name="preference_cdp_summary">Configuration pour l’envoi et la réception</string>
```

File: Nearby Sharing Windows/Resources/values-fr/strings.xml [hunk 4]
```diff
@@ -62,9 +88,12 @@
...
 	<string name="share_file">Partager un fichier</string>
 	<string name="share_url">Partager un lien</string>
+	<string name="share_text">Partager du text</string>
 
 	<string name="send_missing_permissions">Des permissions requises sont manquantes !</string>
-	<string name="wait_for_acceptance">En attente d’acceptation</string>
-	<string name="sending_template">Envoi en cours {0} fichiers</string>
+	<string name="wait_for_acceptance">En attente d’acceptation...</string>
+	<string name="wait_for_acceptance">Annulé</string>
+	<string name="wait_for_acceptance">Terminé</string>
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-fr/strings.xml:96]: duplicate of `wait_for_acceptance`

**Review by ShortDevelopment — CHANGES_REQUESTED** [review]
Hey, thank you very much for your contribution!

There are some minor things to fix:
- The compiler does not allow unescaped apostrophs `'` in xml
You can replace them with `’` or escape them with `\'`
- "Windows" has been removed from the app name for clarity

**Comment by MaisClement** [comment]
Hi,
I'm taking your review into consideration, let me know if there is anything else wrong.

**Comment by 5T748CowJuy** [comment]
Thank you.. I will need to look up translations moving forward.

**Pull request merged**